classes/class-wc-connect-stripe.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Stripe' ) ) {
+
+	class WC_Connect_Stripe {
+
+		/**
+		 * @var WC_Connect_API_Client
+		 */
+		private $api;
+
+		/**
+		 * @var WC_Connect_Options
+		 */
+		private $options;
+
+		/**
+		 * @var WC_Connect_Logger
+		 */
+		private $logger;
+
+		public function __construct( WC_Connect_API_Client $client, WC_Connect_Options $options, WC_Connect_Logger $logger ) {
+			$this->api = $client;
+			$this->options = $options;
+			$this->logger = $logger;
+		}
+
+		public function get_settings( $return_url ) {
```

Comment:
Without a local UI, should this just be an OAuth init URL retrieval method to support API consumers?

Reply:
True, that ca be deleted. If we need it back, it should anyways be implemented in a different function.